ui/src/app/applications/components/applications-list/applications-list.tsx

```diff
@@ -41,7 +42,7 @@ const APP_FIELDS = [
 const APP_LIST_FIELDS = ['metadata.resourceVersion', ...APP_FIELDS.map(field => `items.${field}`)];
 const APP_WATCH_FIELDS = ['result.type', ...APP_FIELDS.map(field => `result.application.${field}`)];
 
-function loadApplications(): Observable<models.Application[]> {
+export const loadApplications = (): Observable<models.Application[]> => {
```

Comment:
I'm wondering if this loadApplications and the related constants above should be moved out of this applications list component, and moved in to the new search-bar component.  That way, both application list and application details will import search-bar.   (Currently, in details, you import from app list, which seems a bit odd).